packages/node-xmpp-server/lib/S2S/Router.js

```diff
@@ -34,9 +34,14 @@ function Router (s2sPort, bindAddress, opts) {
   }
 
   // start tcp socket
-  net.createServer(function (socket) {
+  var server = net.createServer(function (socket) {
     this.acceptConnection(socket)
   }.bind(this)).listen(s2sPort || 5269, bindAddress || '::')
+
+  this.close = function (callback) {
+    debug('closing server')
```

Comment:
it closes the server off to new incoming connections. It doesn't affect the outgoing connection nor existing [incoming connections](https://nodejs.org/api/net.html#net_server_close_callback). Maybe `closeServer` would be a better name?